osgar/record.py

```diff
@@ -52,13 +52,25 @@ def finish(self):
             module.join()
 
 
-def record(config_filename, duration_sec, log_prefix, note=''):
-    log = LogWriter(prefix=log_prefix, note=str(sys.argv))
-    config = load(config_filename)
+def record(config_filename, log_prefix, note, duration_sec=None, application=None):
+    log = LogWriter(prefix=log_prefix, note=note)
+    if type(config_filename) == list:
+        config = load(*config_filename)
+    else:
+        config = load(config_filename)
```

Comment:
Can we do this test the other way around? First test if `config_filename` is string. The other option is more complicated - any iterable of strings...